services-js/access-boston/src/server/access-boston.ts

```diff
@@ -11,7 +11,7 @@ import yar from 'yar';
 import cleanup from 'node-cleanup';
 import acceptLanguagePlugin from 'hapi-accept-language2';
 import hapiDevErrors from 'hapi-dev-errors';
-import next from 'next';
+const next = require('next');
```

Comment:
Hadn't seen require and import mixed in the same file before

Reply:
You know, I was following Fiona’s lead with the PR she made in preparation for this, but I didn’t actually test whether this syntax is necessary anymore... I’ll do so and update the PR.

Reply:
Ok, no prob. I don't see it as it blocker if it's working. Just had the Q 💃